static/js/vue-cdr-access/tests/fixtures/recordListFixture.js

```diff
@@ -76,8 +76,6 @@ export const records_list = [
         "datastream": [
             "techmd_fits|text\/xml|aaa66f91-4870-4937-b7ba-06b015959e4f.xml|xml|5480|urn:sha1:82c71472051b8279a0fbaa537a340c57e3d842f6|aaa66f91-4870-4937-b7ba-06b015959e4f",
             "original_file|image\/png|Screen+Shot+2018-10-17+at+3.02.53+PM.png|png|232738|urn:md5:f5397b230bb5dfa4d53f57ad35514405|aaa66f91-4870-4937-b7ba-06b015959e4f",
-            "thumbnail_small|image\/png|aaa66f91-4870-4937-b7ba-06b015959e4f.png|png|2454||aaa66f91-4870-4937-b7ba-06b015959e4f",
```

Comment:
this should probably have a jp2 instead, with the `|aaa66f91-4870-4937-b7ba-06b015959e4f` owner part at the end